fbw-a32nx/src/systems/fmgc/src/navigation/Navigation.ts

```diff
@@ -79,7 +79,7 @@ export class Navigation implements NavigationProvider {
         facility: null,
     }));
 
-    constructor(private flightPlanService: FlightPlanService, private readonly facLoader: FacilityLoader) {
+    constructor(private flightPlanService: FlightPlanService, private readonly facLoader?: FacilityLoader) {
```

Comment:
Why can `facLoader` be optional?

Reply:
As far as I can see, it's currently not used in this class?

Reply:
You're right. Just pushed that in the latest commit.